sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clienttelemetry/ClientTelemetrySerializer.java

```diff
@@ -47,6 +47,12 @@ public void serialize(ClientTelemetryInfo telemetry, JsonGenerator generator, Se
                 telemetry.getAcceleratedNetworking().toString());
         }
 
+        if (telemetry.getPreferredRegions() != null && telemetry.getPreferredRegions().size() > 0) {
+            generator.writeStringField("preferredRegions",
+                telemetry.getPreferredRegions().toString());
```

Comment:
we should try to normalize the preferredRegions before logging it in the ClientTelemetry, otherwise we may have many different values,

toLowerCase and trim space, this is the same logic that the internal of the SDK uses as well for normalizing the preferred regions

"East US 2" -> "eastus2"

Reply:
We are keeping everything from customer point of view , so we should use exact what they passing